docs/src/Common/MoistThermodynamics.md

```diff
@@ -23,7 +23,7 @@ There are several types of functions:
     * `internal_energy`
     * `air_temperature`
 6. Functions to compute temperatures and partitioning of water into phases in thermodynamic equilibrium (when Gibbs' phase rule implies that the entire thermodynamic state of moist air, including the liquid and ice specific humidities, can be calculated from the 3 thermodynamic state variables, such as energy, pressure, and total specific humidity)
-    * `liquid_fraction_equil` (fraction of condensate that is liquid)
+    * `liquid_fraction` (fraction of condensate that is liquid)
```

Comment:
You might want to update this README more broadly. It has the old arguments I had started out with in the examples, rather than your new thermodynamic state.

Reply:
Raised #568 to fix. I'll write a separate draft PR for you to review.